src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -165,6 +167,10 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
     } yield ()
   }
 
+  def getSynchronizedDate(groupId: WorkbenchGroupIdentity): Future[Option[Date]] = {
+    directoryDAO.getSynchronizedDate(groupId)
```

Comment:
Choosing to pass through to DirectoryDAO from here rather than add DirectoryDAO as a dependency to GoogleExtensionRoutes.